src/Mod/PartDesign/App/FeatureHole.cpp

```diff
@@ -512,6 +512,36 @@ const char* Hole::ThreadSize_ISOmetricfine_Enums[]   = {
     "M100x2.0",    "M100x3.0",    "M100x4.0",    "M100x6.0", NULL };
 const char* Hole::ThreadClass_ISOmetricfine_Enums[]  = { "4G", "4H", "5G", "5H", "6G", "6H", "7G", "7H","8G", "8H", NULL };
 
+// source: https://www.accu.co.uk/en/p/134-iso-metric-thread-tolerances [retrived: 2021-01-11]
+const double Hole::ThreadClass_ISOmetric_data[24][3] = {
+//  Pitch    H      G
+    {0.2,   0.0,   0.017},

```

Comment:
The H column should be omitted. The idea of the H (and also of the h for outer threads) is that it is based on zero and having a bunch of zeroes in the code has no benefit.

Besides this, could you please merge the different commits into one for a better overview?

Reply:
Makes sense. Only kept it because the table is given like this in the standard. With the H column filled out.